src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write code here
+const allLi = document.querySelectorAll('li');
```

Comment:
It's not a good practice to use `document.querySelectorAll('li')` without specifying a parent. This will select all `li` elements on the page, which can potentially lead to unexpected behavior if there are other `li` elements on the page outside of the tree. It would be better to select only the `li` elements within the tree. Consider changing this to `document.querySelector('.tree').querySelectorAll('li')`.